pom.xml

```diff
@@ -260,6 +260,13 @@ under the License.
                 <version>${flink.version}</version>
                 <scope>test</scope>
             </dependency>
+
+            <dependency>
+                <!-- pin transitive dependency of org.apache.hadoop:hadoop-common:2.8.5 -->
+                <groupId>net.minidev</groupId>
+                <artifactId>json-smart</artifactId>
+                <version>2.3</version>
```

Comment:
Use 2.3.1?

Reply:
Fixed in https://github.com/apache/flink-table-store/pull/454/commits/32231caacd7cf6297700049ca39f7816486a1424.